### PR TITLE
Add extraEnv to Management Agent, hostPath ownership change support, update agent image to 1.6.0, update metrics-server to 0.7.2

### PR DESCRIPTION
**Pull request by prasebha**

### CODE REVIEW

**Comment by prasebha** [comment]
waiting on 1.5.0 container image.

**Comment by prasebha** [comment]
waiting for 1.6 image

**Review by niravg75 — APPROVED** [review]

**Review by palukuri — APPROVED** [review]

File: charts/mgmt-agent/values.yaml [hunk 1]
```diff
@@ -78,7 +85,13 @@ deployment:
    
    daemonSet:
       # Provide the host path if Agent is deployed as DaemonSet. Management Agent Pod should have read-write access to it.
+      # Note: The hostPath cannot point to /opt/oracle, please provide another location or create a sub-directory under /opt/oracle and use that as hostPath.
+      # The Management Agent helm chart will attempt to change ownership and permissions on the host path provided here.
```
Comment (santhoshkvuda) [inline charts/mgmt-agent/values.yaml:89]: This comment says, ownership and permission will be modified for the path given here but the command in initContainer has hardcoded value to /opt/oracle. Either of these has to be corrected ?

File: charts/mgmt-agent/README.md [hunk 2]
```diff
@@ -42,6 +43,7 @@ A Helm chart for collecting Kubernetes Metrics using OCI Management Agent into O
 | deployment.cleanupEpochTime | integer | `nil` | Please provide the current epoch time in seconds (Eg: Executing the following command in a bash shell will provide the epoch time: "date +%s") to clean up the agent installation directory from previous deployment |
 | deployment.daemonSetDeployment | bool | `false` | Setting the daemonset deployment to true, will deploy the Management Agents as a daemonset in addition to deploying the Management Agent as a statefulset. This is done to to distribute the node metrics collection to agents running on the node |
 | deployment.daemonSet.hostPath | string | `nil` |  The host path to store data, if Agent is deployed as DaemonSet. Management Agent Pod should have read-write access to it |
+| deployment.daemonSet.overrideOwnership | bool | `true` |  Override the ownership and permissions on the hostPath. The hostPath will be owned by the runAsUser and runAsGroup provided under security context and the permission as 750. </br>Note: This requires oraclelinux:8-slim image </br></br>Setting overrideOwnership to false will disable the ownership change. |
```
Comment (santhoshkvuda) [inline charts/mgmt-agent/README.md:46]: In the note, should we also mention about the initContainer that runs in privileged mode ?

Comment (prasebha) [reply charts/mgmt-agent/README.md:46]: initContainer for change of ownership should always be executing as root (security context 0) to take care of hostPath owned by root as well. this is implicit.

**Review by santhoshkvuda — APPROVED** [review]

**Pull request merged**